src/mumble/os_win.cpp

```diff
@@ -196,8 +196,9 @@ FARPROC WINAPI delayHook(unsigned dliNotify, PDelayLoadInfo pdli) {
 	return 0;
 }
 
+decltype(__pfnDliNotifyHook2) __pfnDliNotifyHook2 = delayHook;
```

Comment:
Matches https://msdn.microsoft.com/en-us/library/f0fkfy9y.aspx I guess...why did the code before work if we actually have to define the symbol ourselves? The decltype is to cope with const and legacy non-const versions of the declaration?